package.json

```diff
@@ -4,6 +4,7 @@
   "description": "In this project I will build a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.",
   "private": "true",
   "type": "module",
+  "main": "index.js",
   "scripts": {
     "watch": "webpack --watch",
     "start": "webpack-dev-server  --config webpack.dev.js --open",
```

Comment:
- [ ] [OPTIONAL] Sometimes, someone may need **to start the webpack dev server** in order to view the website in development mode so It would be better if you add the correct instructions here in the `start` command. For me, I used a live server extension to review your application. 😃 👍